portfolio/src/main/java/com/google/sps/servlets/ServeCommentsServlet.java

```diff
@@ -45,19 +66,52 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String email = request.getParameter("email");
     String content = request.getParameter("content");
-    long timestamp = System.currentTimeMillis();
+    String userTokenId = extractUserTokenId(request);
+
+    try {
+      GoogleIdToken idToken = validateUserIdToken(userTokenId);
+      System.out.println("Token ID is valid.");
```

Comment:
nit: `println`s are great while doing quick debugging, but if they are to remain in the code, they should be converted to some real-world logging

Reply:
Is there any recommended way to do real-world logging -- should I use java.util.logging or something different? 
https://stackoverflow.com/questions/11359187/why-not-use-java-util-logging

Reply:
https://cloud.google.com/appengine/docs/standard/java/logs

Reply:
For now, I'll just remove it and add an issue to add some logging.